test/re/frida/MethodTest.java

```diff
@@ -39,6 +40,99 @@ public void replacementPropagatesExceptions() {
         thrown.expectMessage("Already dead");
         badger.die();
     }
+    
+    static private String failString = null;
+    static private void Fail( String msg )
+    {
+      failString = msg;
+    }
+    
+    @Test
+    public void TestNewInterface() {
+        loadScript("var X509TrustManager = Java.use('javax.net.ssl.X509TrustManager');" +
+                "try{" +
+                "  var tm = X509TrustManager.$new();" +
+                "}catch(e){" + 
+                "  var MethodTest = Java.use('re.frida.MethodTest');" +
+                "  MethodTest.Fail('couldnt create trustmanager: ' + e);" +
```

Comment:
Let's use `script.getNextMessage()` instead (like above), that way we `send('some string');` and assert that `script.getNextMessage()` gives us the same string.